examples/arm/executor_runner/CMakeLists.txt

```diff
@@ -59,8 +59,74 @@ endif()
 
 if(TARGET_BOARD STREQUAL "corstone-300")
   add_subdirectory(${ETHOS_SDK_PATH}/core_platform/targets/corstone-300 target)
+  target_compile_definitions(ethosu_target_common INTERFACE
```

Comment:
Nit: add a comment that this is from ethos-u-core and the common target to link against for the apps.